### DISCUSSION TITLE
Can I use Recoil JS for production?

### DISCUSSION THREAD

**Discussion by sultondev** [facebookexperimental/Recoil#1746] [Q&A]
I started thinking about using Recoil in real projects when I switched to Recoil from Redux. I love Recoil JS. My company gave a task. I have to create a marketplace. I'm here to ask you. Can I use Recoil JS for production and won't I have problems with that in the future?

**Comment by BenjaBobs** [comment]
My team uses RecoilJS in production for multiple SPA's, and use it to handle almost all our state.
We're quite happy with the results.

See also:
- https://github.com/facebookexperimental/Recoil/issues/691
- https://github.com/facebookexperimental/Recoil/issues/1495

**Reply by sultondev** [reply]
Thank you very much 😊
